pkgs/development/compilers/spirv-llvm-translator/default.nix

```diff
@@ -30,6 +30,14 @@ stdenv.mkDerivation rec {
   # FIXME: CMake tries to run "/llvm-lit" which of course doesn't exist
   doCheck = false;
 
+  postBuild = ''
+    make llvm-spirv
```

Comment:
I think it would be cleaner to set
```nix
makeFlags = [ "all" "llvm-spirv" ];
```

Rather than adding a `postBuild` phase, but it's a small thing.